Tasks/AzureFunctionOnKubernetesV1/src/clusterconnection.ts

```diff
@@ -60,14 +61,24 @@ export default class ClusterConnection {
             kubeconfig = await this.getKubeConfig(connectionType);
         }
 
-        return this.initialize().then(() => {
+        return this.initialize().then(async () => {
             if (kubeconfig)
             {
                 this.kubeconfigFile = path.join(this.userDir, "config");
                 fs.writeFileSync(this.kubeconfigFile, kubeconfig);
             }
 
             process.env["KUBECONFIG"] = this.kubeconfigFile;
+
+            try {
+              const kubelogin = new Kubelogin(this.userDir);
+              if (kubelogin.isAvailable()) {
+                tl.debug('Kubelogin is installed. Converting kubeconfig.');
+                await kubelogin.login(tl.getInput('azureSubscriptionEndpoint', false));
+              }
+            } catch (err) {
+              tl.warning(tl.loc('KubeloginFailed', err));
```

Comment:
>warning [](http://example.com/codeflow?start=17&length=7)

same feedback on error warning handling as Tasks/KubernetesV1/src/clusterconnection.ts

Reply:
change to debug message here too